eng/pipelines/templates/jobs/mgmt-release.yml

```diff
@@ -0,0 +1,102 @@
+
+        strategy:
+          runOnce:
+            deploy:
+              steps:
+                - checkout: self
+                - checkout: azure-sdk-build-tools
+
+                - download: current
+                  artifact: packages
+                  displayName: Download published Nuget Packages
+
+                - task: Powershell@2
+                  displayName: 'Verify Package Tags and Create Git Releases'
+                  inputs:
+                    filePath: ${{ parameters.BuildToolsPath }}/scripts/create-tags-and-git-release.ps1
+                    arguments: >
+                      -artifactLocation '$(PIPELINE.WORKSPACE)/packages'
+                      -workingDirectory $(System.DefaultWorkingDirectory)
+                      -packageRepository 'Nuget'
+                      -releaseSha '$(Build.SourceVersion)'
+                      -repoOwner 'azure-sdk-for-net'
+                      -repoName 'Azure'
+                    pwsh: true
+                  env:
+                    GH_TOKEN: $(azuresdk-github-pat)
+                  condition: and(succeeded(), ne(variables['SkipCreateTagAndGitRelease'], 'true'))
+
+                - ${{ if eq(parameters.ShouldPublish, true) }}:
+                  - template: pipelines/steps/net-signing.yml@azure-sdk-build-tools
+                    parameters:
+                      PackagesPath: '$(PIPELINE.WORKSPACE)/packages'
+                      BuildToolsPath: '${{ parameters.BuildToolsPath }}'
+
+                  - task: MSBuild@1
+                    displayName: 'Upload Symbols'
+                    inputs:
+                      solution: '${{ parameters.BuildToolsPath }}/tools/symboltool/SymbolUploader.proj'
+                      msbuildArguments: >
+                        /p:PackagesPath='$(PIPELINE.WORKSPACE)/packages'
+                        /p:MSPublicSymbolsPAT=$(azuresdk-microsoftpublicsymbols-devops-pat)
+                        /p:MSSymbolsPAT=$(azuresdk-microsoft-devops-pat)
+                        /p:AzureSDKSymbolsPAT=$(azuresdk-azure-sdk-devops-pat)
+
+                  - task: NuGetToolInstaller@1
+                    displayName: 'Use NuGet ${{ parameters.NugetVersion }}'
+                    inputs:
+                      versionSpec: ${{ parameters.NugetVersion }}
+
+                  - task: NuGetCommand@2
+                    displayName: 'Publish to Nuget'
+                    inputs:
+                      command: push
+                      packagesToPush: '$(PIPELINE.WORKSPACE)/packages/**/*.nupkg;!$(PIPELINE.WORKSPACE)/packages/**/*.symbols.nupkg'
+                      nuGetFeedType: external
+                      publishFeedCredentials: Nuget.org
+
+                  - task: AzureFileCopy@2
+                    displayName: 'Copy Signed Files to Blob'
+                    inputs:
+                      SourcePath: '$(PIPELINE.WORKSPACE)/packages'
+                      azureSubscription: '${{ parameters.PartnerDropsSubscription }}'
```

Comment:
I suspect this is not going to work. We should also consider whether or not we want to publish to a blob container at all or just publish them as build artifacts. I'd suggest the later.

Reply:
Am not sure why they do this publish honestly, it's probably not necessary but it did work in my test runs, I had to add the secretes to the mgmt pipeline though

Reply:
Yes the secret access was the think I was worried about. They used to do this because release pipelines didn't have a notion of artifacts. So perhaps we should just switch to artifacts here and update any docs pointing at this blob storage.